### PR TITLE
starrocks helm:enable capabilities in starrocksBeSpec

### PR DESCRIPTION
**Pull request by mingmxu**
manual test as below:

# empty capabilities
Screenshot 2024-01-17 at 9 20 38 PM

# with capabilities
Screenshot 2024-01-17 at 9 19 48 PM

### CODE REVIEW

**Comment by mingmxu** [comment]
pending on yaml validation agains CRD

File: helm-charts/charts/kube-starrocks/charts/starrocks/templates/starrockscluster.yaml
Comment (yandongxiao) [inline helm-charts/charts/kube-starrocks/charts/starrocks/templates/starrockscluster.yaml:1]: Does FE/CN need the capabilities ?

File: helm-charts/charts/kube-starrocks/charts/starrocks/values.yaml
Comment (yandongxiao) [inline helm-charts/charts/kube-starrocks/charts/starrocks/values.yaml:1]: should execute `bash create-parent-chart-values.sh` in `scripts` directory to update values.yaml of kube-starrocks chart.

File: helm-charts/charts/kube-starrocks/charts/starrocks/values.yaml [hunk 1]
```diff
@@ -404,6 +404,14 @@ starrocksBeSpec:
...
   # The userId will be set to 1000, and the groupID will be set to 1000.
   runAsNonRoot: false
+  # add/drop capabilities for BE container, DataDog profiling lib relies on it for example. It could be added as below:
+  # capabilities:
+  #    add:
+  #      - PERFMON
+  #      - SYS_PTRACE
+  #    drop:
+  #      - SYS_ADMIN
+  capabilities: {}
```
Comment (yandongxiao) [inline helm-charts/charts/kube-starrocks/charts/starrocks/values.yaml:414]: Can you please add the example comment below `capabilities ` like other fileds ?

File: helm-charts/charts/kube-starrocks/charts/starrocks/templates/starrockscluster.yaml
Comment (mingmxu) [reply helm-charts/charts/kube-starrocks/charts/starrocks/templates/starrockscluster.yaml:1]: not as I know, only BE is required

**Review by yandongxiao — APPROVED** [review]

**Comment by yandongxiao** [comment]
thank you for your contribution!

**Pull request merged**